Examples/test-suite/javascript/using2_workers_runme.js

```diff
@@ -0,0 +1,11 @@
+try {
+    const { Worker } = require("worker_threads");
+
+    const worker1 = new Worker("./using2_runme.js");
+    const worker2 = new Worker("./using2_runme.js");
+
+    worker1.on('exit', (code) => { });
+    worker2.on('exit', (code) => { });
+} catch(err) {
```

Comment:
This swallows all exceptions, so any problems in the test get masked for all versions of Javascript. Seems hardly worth having the runtime test! Can you not detect the supported version at runtime and then only test.

Reply:
> Can you not detect the supported version at runtime and then only test.

The source of my hesitation to do so is that I didn't want to make it node-specific. I mean node version detection is node-specific, and I figured that it won't be appreciated, because the scripts are exercised with other javascript engines. Or do you mean detect node version in make rule? This of course can be done... Even though workers made their appearance in 10.5, it was 12.0 that shipped them as integral part of the run-time environment.

Reply:
> I didn't want to make it node-specific.

I think I've conflated unrelated thoughts here. I mean given the **presented** suggestion we already know that the script in question is executed exclusively by node. Hence it doesn't have to be engine-neutral. But I did think about it initially, and apparently failed to disregard the thought when commenting. So that yes, it's totally possible to detect if we are running on node v12+ and perform test only then.

I'll wait for response to the question if "side-kick" `$*_workers$(SCRIPTSUFFIX)` is generally acceptable before I proceed.